chart/templates/redis/redis-statefulset.yaml

```diff
@@ -80,7 +79,7 @@ spec:
         - name: redis
           image: {{ template "redis_image" . }}
           imagePullPolicy: {{ .Values.images.redis.pullPolicy }}
-          securityContext: {{ $containerSecurityContext | nindent 12 }}
+          securityContext: {{- toYaml .Values.redis.securityContexts.container | nindent 12 }}
```

Comment:
This breaks backcompat on `redis.securityContext`.

Reply:
Added the same change as with stastd.securityContext